### ISSUE TITLE
stuck when download CC3M 

### ISSUE THREAD

**Issue by jihwanp** [facebookresearch/Detic#28]
Hello, I have a question about downloading the CC3M image. 
At the beginning of the download using `download_cc.py`, it stucked by logging `proxy request sent, waiting response`, and although I use the timeout option in wget, it keeps hanging.
I wonder if it occurs in my case or if it's a phenomenon that other people are experiencing.

Below is the log where I face the hanging.
```
--2022-02-19 02:42:23--  http://www.mercedsunstar.com/news/9d6aao/picture82035257/alternates/FREE_640/13330875_1110997995625119_2134033517544198418_n
Resolving bc-proxy.devel.kakao.com (bc-proxy.devel.kakao.com)... 10.41.244.126
Connecting to bc-proxy.devel.kakao.com (bc-proxy.devel.kakao.com)|10.41.244.126|:13128... connected.
Proxy request sent, awaiting response... 301 Moved Permanently
Location: https://www.mercedsunstar.com/news/9d6aao/picture82035257/alternates/FREE_640/13330875_1110997995625119_2134033517544198418_n [following]
--2022-02-19 02:42:23--  https://www.mercedsunstar.com/news/9d6aao/picture82035257/alternates/FREE_640/13330875_1110997995625119_2134033517544198418_n
Connecting to bc-proxy.devel.kakao.com (bc-proxy.devel.kakao.com)|10.41.244.126|:13128... connected.
Proxy request sent, awaiting response... 
```

**Comment by xingyizhou** [comment]
Hi,

  Thank you for raising up this issue. This script is not fully tested when we released the code. We used a pre-downloaded CC dataset on our cluster during development. 

  A simple workaround would be using `'wget {} --timeout 1 -t 1 -O {}/{}.jpg'.format(path, args.save_image_path, i + 1)` for downloading. However on my machine, it still needs more than a week to download. 

  Another workaround would be using a third-party CC downloader (e.g., [this](https://huggingface.co/spaces/flax-community/dalle-mini/commit/75b01a0a3a29bb2eb6962f5f2fdf160e5c784647)), and converting it to our format (make sure the image file name is the line number in the tsv file, and have a `caption` field being a list of a single element of the caption). We'll test this later.

Best,
Xingyi
